integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/distrox/DistroXMarketplaceImageTests.java

```diff
@@ -0,0 +1,88 @@
+package com.sequenceiq.it.cloudbreak.testcase.e2e.distrox;
+
+import static com.sequenceiq.it.cloudbreak.context.RunningParameter.key;
+import static java.lang.String.format;
+
+import java.util.concurrent.atomic.AtomicReference;
+
+import javax.inject.Inject;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testng.annotations.Test;
+
+import com.sequenceiq.it.cloudbreak.client.DistroXTestClient;
+import com.sequenceiq.it.cloudbreak.client.ImageCatalogTestClient;
+import com.sequenceiq.it.cloudbreak.cloud.v4.CloudProvider;
+import com.sequenceiq.it.cloudbreak.context.Description;
+import com.sequenceiq.it.cloudbreak.context.TestContext;
+import com.sequenceiq.it.cloudbreak.dto.distrox.DistroXTestDto;
+import com.sequenceiq.it.cloudbreak.dto.distrox.image.DistroXImageTestDto;
+import com.sequenceiq.it.cloudbreak.dto.imagecatalog.ImageCatalogTestDto;
+import com.sequenceiq.it.cloudbreak.exception.TestFailException;
+import com.sequenceiq.it.cloudbreak.log.Log;
+import com.sequenceiq.it.cloudbreak.testcase.e2e.AbstractE2ETest;
+import com.sequenceiq.it.cloudbreak.util.spot.UseSpotInstances;
+
+public class DistroXMarketplaceImageTests extends AbstractE2ETest {
+    private static final Logger LOGGER = LoggerFactory.getLogger(DistroXMarketplaceImageTests.class);
+
+    @Inject
+    private DistroXTestClient distroXTestClient;
+
+    @Inject
+    private ImageCatalogTestClient imageCatalogTestClient;
+
+    @Override
+    protected void setupTest(TestContext testContext) {
+        createDefaultUser(testContext);
+        createDefaultCredential(testContext);
+        initializeDefaultBlueprints(testContext);
+        createDefaultDatalake(testContext);
+    }
+
+    @Test(dataProvider = TEST_CONTEXT)
+    @UseSpotInstances
+    @Description(
+            given = "there is a running cloudbreak",
+            when = "a valid DistroX create request is sent (latest Marketplace Image)",
+            then = "DistroX should be available and deletable")
+    public void testDistroXWithMarketplaceImageCanBeCreatedSuccessfully(TestContext testContext) {
+        String imageSettings = resourcePropertyProvider().getName();
+        String distrox = resourcePropertyProvider().getName();
+        AtomicReference<String> selectedImageID = new AtomicReference<>();
+        CloudProvider cloudProvider = testContext.getCloudProvider();
+
+        String imgCatalogKey = "mp-img-cat";
+        testContext
+                .given(imgCatalogKey, ImageCatalogTestDto.class)
+                .withName(imgCatalogKey)
+                .withUrl("https://gist.githubusercontent.com/daszabo/6b53c1a413d9b6cb847b9ad786e7bbf5/raw/0edb66302b7f485527b3bc03fdff1d68695d354c/" +
```

Comment:
Would be better to have an official marketplace image catalog, up in s3, next to the test image catalog.